.devcontainer/devcontainer.json

```diff
@@ -7,6 +7,8 @@
 	"runArgs": [ "--privileged" ],
 	// start redis-server and copy launch profiles to .vscode
 	"postCreateCommand": "/etc/init.d/redis-server start && mkdir -p .vscode && cp docker/vscode/vscode-launch.json .vscode/launch.json",
```

Comment:
Remove the starting of the redis-server from the postCreateCommand as it will always be started with the postStartCommand.

Reply:
Oh okay will do that!

Reply:
Hey @hacktobeer 
I have squashed the commits, could you please approve the workflow again.